README.md

```diff
@@ -45,59 +45,11 @@ Then you can simply go to either of the following addresses in your browser to a
 
 If you face problems while installing Jekyll or using its gem bundler you may go through its [troubleshooting docs](https://jekyllrb.com/docs/troubleshooting/)
 
```

Comment:
Should there be a reference to the example markdown file here ?
(Something like - "A sample project definition is availaible here (with a hyperlink)" ,Since this example file won't be visible on the projects page and is also not referred anywhere else, it might get hard for people to find the example.md)